### PR TITLE
COR-44  Applying theming to story cards, COR-8   Story Card Text Embe…

### PR DESCRIPTION
**Pull request by stringtheory**
…llishment Classes

X COR-44  Applying theming to story cards
X COR-8   Story Card Text Embellishment Classes

### CODE REVIEW

**Comment by stringtheory** [comment]
COR-44  Applying theming to story cards
COR-8   Story Card Text Embellishment Classes

File: src/components/Contentful/CardRow.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			#content class="tw-bg-primary"
+		>
+			<kv-page-container>
+				<kv-grid
+					class="tw-mx-auto tw-grid-cols-12"
+				>
+					<div
+						v-for="(storyCard, index) in storyCards"
+						:key="index"
+						:class="`tw-col-span-12 md:tw-col-span-${12/storyCards.length}`"
```
Comment (BoulderBrains) [inline src/components/Contentful/CardRow.vue:16]: I tried to do something l similar in a recent PR. Once the code got up to development the tailwind class didn't work due to the tw-class name being split. We went with this method instead: 
:class="{
							'tw-col-span-12': true,
							'md:tw-col-span-6': contentfulMediaItems.length >= 2,
							'lg:tw-col-span-6': contentfulMediaItems.length === 2,
							'lg:tw-col-span-4': contentfulMediaItems.length === 3,
							'lg:tw-col-span-3': contentfulMediaItems.length === 4,
						}"
That way complete tailwind classes are pass through and respected on render.

Comment (mcstover) [reply src/components/Contentful/CardRow.vue:16]: That's a good call. It's surprisingly working now... Probably b/c of your code lolz. I'll update with the more specific classes.

File: src/components/Contentful/StoryCard.vue [hunk 2]
```diff
@@ -1,58 +1,78 @@
...
+			/>
+			<dynamic-rich-text class="tw-text-center" :html="footer" />
+		</div>
+	</kv-theme-provider>
 </template>
 
 <script>
 import { richTextRenderer } from '@/util/contentful/richTextRenderer';
 import DynamicRichText from '@/components/Contentful/DynamicRichText';
 import KvContentfulImg from '@/components/Kv/KvContentfulImg';
```
Comment (eddieferrer) [inline src/components/Contentful/StoryCard.vue:60]: Please swap this out with the component in ui elements:
`import KvContentfulImg from '~/@kiva/kv-components/vue/KvContentfulImg';`
There should be no difference in functionality but we will eventually deprecate '@/components/Kv/KvContentfulImg'

File: src/components/Contentful/StoryCard.vue [hunk 3]
```diff
@@ -63,23 +83,39 @@ export default {
 			default: () => {},
 		},
 	},
+	data() {
```
Comment (ryan-ludwig) [inline src/components/Contentful/StoryCard.vue:86]: Minor, since they're referenced in `computed` below, these shouldn't be needed in `data` as well

**Review by mcstover — APPROVED** [review]
Thank you @stringtheory for the very strong foundation with these components! I made a few tweaks but it's generally in order.

**Pull request merged**